plugins/auth-backend/src/providers/microsoft/provider.ts

```diff
@@ -14,218 +14,25 @@
-    });
-  }
-
-  async handler(req: express.Request) {
-    const { result, privateInfo } = await executeFrameHandlerStrategy<
-      OAuthResult,
-      PrivateInfo
-    >(req, this._strategy);
-
-    return {
-      response: await this.handleResult(result),
-      refreshToken: privateInfo.refreshToken,
-    };
-  }
-
-  async refresh(req: OAuthRefreshRequest) {
-    const { accessToken, refreshToken, params } =
-      await executeRefreshTokenStrategy(
-        this._strategy,
-        req.refreshToken,
-        req.scope,
-      );
-
-    return {
-      response: await this.handleResult({
-        params,
-        accessToken,
-        ...(!this.skipUserProfile(accessToken) && {
-          fullProfile: await executeFetchUserProfileStrategy(
-            this._strategy,
-            accessToken,
-          ),
-        }),
-      }),
-      refreshToken,
-    };
-  }
-
-  private async handleResult(result: {
-    fullProfile?: PassportProfile;
-    params: {
-      id_token?: string;
-      scope: string;
-      expires_in: number;
-    };
-    accessToken: string;
-    refreshToken?: string;
-  }): Promise<OAuthResponse> {
-    let profile = {};
-    if (result.fullProfile) {
-      const photo = await this.getUserPhoto(result.accessToken);
-      result.fullProfile.photos = photo ? [{ value: photo }] : undefined;
-      ({ profile } = await this.authHandler(
-        result as OAuthResult,
-        this.resolverContext,
-      ));
-    }
-
-    const expiresInSeconds =
-      result.params.expires_in === undefined
-        ? BACKSTAGE_SESSION_EXPIRATION
-        : Math.min(result.params.expires_in, BACKSTAGE_SESSION_EXPIRATION);
-
-    return {
-      providerInfo: {
-        accessToken: result.accessToken,
-        scope: result.params.scope,
-        expiresInSeconds,
-        ...{ idToken: result.params.id_token },
-      },
-      profile,
-      ...(result.fullProfile &&
-        this.signInResolver && {
-          backstageIdentity: await this.signInResolver(
-            { result: result as OAuthResult, profile },
-            this.resolverContext,
-          ),
-        }),
-    };
-  }
-
-  private async getUserPhoto(accessToken: string): Promise<string | undefined> {
-    try {
-      const res = await fetch(
-        'https://graph.microsoft.com/v1.0/me/photos/48x48/$value',
-        {
-          headers: {
-            Authorization: `Bearer ${accessToken}`,
-          },
-        },
-      );
-      const data = await res.buffer();
-
-      return `data:image/jpeg;base64,${data.toString('base64')}`;
-    } catch (error) {
-      this.logger.warn(
-        `Could not retrieve user profile photo from Microsoft Graph API: ${error}`,
-      );
-      return undefined;
-    }
-  }
-}
+  microsoftAuthenticator,
+  microsoftSignInResolvers,
+} from '@backstage/plugin-auth-backend-module-microsoft-provider';
 
 /**
- * Auth provider integration for Microsoft auth
+ * Auth provider integration for GitLab auth
```

Comment:
```suggestion
 * Auth provider integration for Microsoft auth
```

Reply:
oh, seems like the reverted commit still had that. As it's merged already, I will create a tiny PR to take care of it.